rules-tests/DeadCode/Rector/Node/RemoveNonExistingVarAnnotationRector/Fixture/skip_used_after_next_statement.php.inc

```diff
@@ -1,29 +0,0 @@
-<?php
-
-namespace Rector\Tests\DeadCode\Rector\Node\RemoveNonExistingVarAnnotationRector\Fixture;
-
-class SkipUsedAfterNextStatement
-{
-    public function get(): void
-    {
-        $options = [
-            'a' => 'value a',
-            'b' => 'value b',
-            'c' => 'value c',
-        ];
-
-        extract($options);
```

Comment:
I can see framework like CakePHP in the past using extract a lot when assign controller variable to view, I think StmtsAwareInterface loop can be used to save existing variable name of array key value via ValueResolver to be verified with variable name can be used to detect it.